WNPRC_Purchasing/src/client/components/DocumentAttachmentPanel.tsx

```diff
@@ -39,12 +39,12 @@ export const DocumentAttachmentPanel: FC<Props> = memo(props => {
     return (
         <>
             <Panel
-                className="panel panel-default"
+                className={'panel panel-default domain-form-panel'}
                 expanded={true}
                 onToggle={function () {}} // this is added to suppress JS warning about providing an expanded prop without onToggle
             >
-                <div className="bg-primary">
-                    <Panel.Heading>
+                <div>
+                    <Panel.Heading className={'bg-primary'}>
```

Comment:
just double checking, was this an intentional change? it is unrelated to the domain-form-panel class change, so wanted to check.